src/roswire/app/instance.py

```diff
@@ -125,6 +125,7 @@ def catkin_make(self, directory: str) -> CatkinMake:
                           files=self.files)
 
     @contextlib.contextmanager
+    # TODO: Make this ros1
```

Comment:
Out of scope for this PR, so let's remove the comment :+1:

Reply:
Done